bindata/v4.0.0/controller/clusterrole.yaml

```diff
@@ -62,3 +62,11 @@ rules:
   - list
   - watch
   - update
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - infrastructures
+  verbs:
+  - get
+  - list
+  - watch
```

Comment:
separate commit, explain in commit message please

Reply:
89f04d8